web/src/components/ManagePlan/ManagePlan.js

```diff
@@ -34,28 +35,29 @@ const ManagePlan = ({ plan }) => {
         <Heading>{plan.name}</Heading>
       </Box>
       <Divider my="5" />
-      <Grid templateColumns="5fr 1fr" gridColumnGap={5}>
-        <Stack>
-          <Grid templateColumns="repeat(7, 1fr)" gridColumnGap={2}>
-            {daysOfWeek.map((dayOfWeek, index) => {
-              return (
-                <Box p="3" bg="green.800" key={`dow-${index}`}>
-                  <Text fontSize="md" color="white" textAlign="center">
-                    {dayOfWeek}
-                  </Text>
-                </Box>
-              )
-            })}
-            <PlanWeeks
-              planWeeks={plan.planWeeks}
-              onWorkoutSelected={(workout) => {
-                setSelectedWorkoutId(workout.id)
-              }}
-            />
-          </Grid>
-        </Stack>
-        <WorkoutDetailCell id={selectedWorkoutId} />
-      </Grid>
+      <Flex alignItems="flex-start">
```

Comment:
I've bounced back and forth between this top-level container being grid vs flex, but it needs to be flex in order for me to use `position: sticky` in a child element.